[0m[[0m[0mdebug[0m] [0m[0m> Exec(run, Some(3bf46aa0-9611-4c02-8691-75a3dca0f129), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Compile / run[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 1 Scala source to /Users/katherine/Desktop/Projecten/FP_exercises/target/scala-3.1.1/classes ...[0m
[0m[[0m[31merror[0m] [0m[0m[31m-- Error: /Users/katherine/Desktop/Projecten/FP_exercises/src/main/scala/ZIO.scala:7:7 [0m[0m
[0m[[0m[31merror[0m] [0m[0m[31m7 |[0m[33mobject[0m [35mZIO[0m [33mextends[0m [35mZIOAppDefault[0m {[0m
[0m[[0m[31merror[0m] [0m[0m  |       [31m^[0m[0m
[0m[[0m[31merror[0m] [0m[0m  |object creation impossible, since def run: [0m
[0m[[0m[31merror[0m] [0m[0m  |  zio.ZIO[ZIOApp.this.Environment & (zio.ZEnv & zio.Has[zio.ZIOAppArgs]), Any, [0m
[0m[[0m[31merror[0m] [0m[0m  |    Any[0m
[0m[[0m[31merror[0m] [0m[0m  |  ] in [33mtrait[0m [35mZIOApp[0m in [33mpackage[0m [35mzio[0m is not defined [0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mcompileIncremental[0m) Compilation failed[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 0 s, completed 10 Jun 2022, 15:22:20[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(shell, None, None)[0m
